src/pybind/cephfs/cephfs.pyx

```diff
@@ -1901,6 +1901,24 @@ cdef class LibCephFS(object):
                           st_mtime=datetime.fromtimestamp(stx.stx_mtime.tv_sec),
                           st_ctime=datetime.fromtimestamp(stx.stx_ctime.tv_sec))
 
+    def is_dir_exists(self, path, str_type=b''):
```

Comment:
Is there a need for having interfaces for each (file) type?

And what is `str_type`? Needs explanation.

Reply:
it's for whether the path string is a char string or byte string. Usually its always byte string but because I saw this kind of configuration somewhere(I think it was cephfs-shell), i thought of providing this option.

Reply:
I think we can just accept byte string as in the rest of the interfaces.

Reply:
@vshankar it's done. PTAL

Reply:
isn't it useful? i mean it would be good to have functions that checks for dir/file or just a broad idea if the path exists or not

Reply:
those interfaces proved not useful as using it directly makes no sense as we need a fs connection handle to carry out `stat`, both of these interfaces have been removed.